spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/data/redis/RedisProperties.java

```diff
@@ -325,6 +326,14 @@ public void setPool(Pool pool) {
 		 */
 		private Pool pool;
 
+		/**
+		 * If shareNativeConnection is true, the pool will be used to select a connection
```

Comment:
That's a bit unusual for the description of a key (see other descriptions around for inspiration). The first sentence must be a concise summary of the purpose of the key as it's the one showing up in auto-completion popup in an IDE.